### DISCUSSION TITLE
Allowing line breaks \n

### DISCUSSION THREAD

**Discussion by robinNavas** [Respect/Validation#1464] [Q&A]
Hi,

I am quite a newbie in fields validation and I am struggling with the validation of a textarea form field. I use the `alnum(" ")` rule but when I add a line break in the field the validation fails because there is \n characters included in the value. I can't figure out how to add them to the alnum rule, I have test `alnum(' \ \n')` with no result.

I wonder what is the best solution to validate this type of data. 

Thank's, and sorry this is certainly a very basic question but I am quite blocked right now.

**Comment by henriquemoody** [comment]
The problem is that `'\n'` as a string in single quote is not a real new line. Have you tried `alnum(' ' . PHP_EOL)`?
